@xen-orchestra/backups/RemoteAdapter.js

```diff
@@ -552,3 +551,17 @@ exports.RemoteAdapter = class RemoteAdapter {
     return Object.defineProperty(JSON.parse(await this._handler.readFile(path)), '_filename', { value: path })
   }
 }
+
+// TODO: implement a lock for the passed dir which will be synchronized with running backups
+const lockDir = dir => () => {}
```

Comment:
This stub does not make a lot of sense, IMHO it would be better as a `fs/RemoteHandler` method, what do you think?

If you agree, I think we should go ahead and implement it in a PR that will be merged before this one.

My proposal:
- `AbractHandler#lock(path)` with a default noop implementation
- `LocalHandler#_lock(path)` which an implementation based on `proper-lockfile`